### DISCUSSION TITLE
Phân công câu hỏi tuần 9

### DISCUSSION THREAD

**Discussion by baonhu1304** [baonhu1304/Lop-10A2-nhom-1#15] [General]
Đầu tuần thầy tặng 4 món quà. Mn dzô lựa chọn "quà" cho mình nha.
Sẽ có 2 bạn cùng làm câu 3 nha. Ai chọn "quà" nào thì nhắn bên dưới giúp tui nhe.
Cảm ơn mn

**Comment by thaopham2501** [comment]
hoy phân công đi :))))))))) cho t làm ít thôi, t giúp cái báo cáo :b

**Reply by baonhu1304** [reply]
thoi để nay thầy triển khai gòi mình tính hen

**Reply by thaopham2501** [reply]
t xin chọn câu 1 :)))))

**Comment by Lenga10A2** [comment]
Ok lun

Vào 09:36, T.5, 4 Th11, 2021 Phạm Bảo Như ***@***.***> đã
viết:

**Comment by Lenga10A2** [comment]
Tui câu 2 lun cho

Vào 16:52, T.5, 4 Th11, 2021 Phạm Ngọc Hương Thảo ***@***.***>
đã viết:

**Comment by baonhu1304** [comment]
Tui câu 3 hen. Ai làm chung hong

**Comment by letu25082006** [comment]
vậy tui làm c3 với Như lun đc hong

**Reply by baonhu1304** [reply]
Ok cám ơn bà nhiều nha. Bà làm cái vế "tạo ra 3 bài toán có sử dụng những công thức toán đã được học" giúp tui được hơm. Ủa mà cái đó có cần làm hong bà???

**Comment by NguyenNgocNhuY** [comment]
@baonhu1304 ơi câu 4 tui làm liệt kê bà làm sơ đồ khối được k

**Reply by baonhu1304** [reply]
Hay bà làm Vd1, tui Vd 2, rồi Vd3 mình chia nhau vậy hen

**Reply by NguyenNgocNhuY** [reply]
oke

**Comment by NguyenNgocNhuY** [comment]
v vd3 tui liệt kê bà sơ đồ khối nghen

**Reply by baonhu1304** [reply]
oke

**Comment by letu25082006** [comment]
ủa mình discuss zị ùi thầy thấy hong ta

**Reply by NguyenNgocNhuY** [reply]
chắc thầy thấy á thầy kêu nên dùng nhiều những phần này mà

**Comment by NguyenNgocNhuY** [comment]
@baonhu1304 cái câu 4 phần sau á tui làm v là đúng chưa ta

**Reply by baonhu1304** [reply]
bà uii, hay là mình ghi cụ thể ra lun hơm

**Reply by NguyenNgocNhuY** [reply]
tui cũng k rõ nữa

**Reply by baonhu1304** [reply]
;((

**Reply by NguyenNgocNhuY** [reply]
câu này thấy có vẻ giống ghi lý thuyết ra á

**Reply by NguyenNgocNhuY** [reply]
v ghi lý thuyết th ha v tui làm r bà khỏi cần làm phần này ha

**Reply by baonhu1304** [reply]
hay mình hỏi thầy thử hong

**Reply by NguyenNgocNhuY** [reply]
oke bà hỏi đi @baonhu1304

**Reply by HienCaoPhuNhuan** [reply]
> hay mình hỏi thầy thử hong

rồi sao ko hỏi

**Reply by baonhu1304** [reply]
Ủa, con hong thấy bạn trả lời. Nay dzô mới thấy lun á Thầy. 
Nhưng mà thôi con thấy bạn làm ổn quá, nên con hong hỏi Thầy lun hihi

**Comment by letu25082006** [comment]
tui nghĩ chắc oki gùi á

**Reply by NguyenNgocNhuY** [reply]
tại tui sợ bị hiểu sai câu hỏi á tks bà nha
